build.sbt

```diff
@@ -32,6 +32,8 @@ val extraDependencies = Seq(
   "org.scalactic" %% "scalactic" % "3.2.14",
   "io.spray" %% "spray-json" % "1.3.5",
   "com.jcraft" % "jsch" % "0.1.54",
+  "com.pauldijou" %% "jwt-core" % "3.0.0",
```

Comment:
we have some code that parses jwt, check the certified events logging code

Reply:
I didn't find any jwt related code, I am reading this: (certified_event)[https://github.com/microsoft/SynapseML/blob/master/core/src/main/scala/com/microsoft/azure/synapse/ml/logging/fabric/CertifiedEventClient.scala], can you point me to the right place